gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/watermark/StringWatermark.java

```diff
@@ -26,19 +26,22 @@
 
 import lombok.AllArgsConstructor;
 import lombok.EqualsAndHashCode;
-import lombok.Getter;
 
 
 /**
  * String based {@link ComparableWatermark} implementation.
  */
 @AllArgsConstructor
 @EqualsAndHashCode
-public class StringWatermark implements ComparableWatermark {
+public class StringWatermark implements ComparableWatermark<String> {
```

Comment:
I don't see any tests that validates this change. How is it verified? Could you help confirm?

Reply:
There is no API change in this class at all. i.e. the getValue still return String type and the way to construct the class still remain the same. So I think we don't need test for this part. Let me know if you still have concerns about this.